src/roles/elasticsearch/templates/elasticsearch.yml.j2

```diff
@@ -41,10 +41,10 @@ path.logs: {{ m_meza_data }}/elasticsearch/log
 # Fixes xpack SSL java access denied at ES startup.
 #
 # Probably not the best way to do this. Suggestions welcome
-xpack.ssl.truststore.path: /etc/elasticsearch/cacerts-java-extracted
-xpack.ssl.truststore.password: changeit
-reindex.ssl.truststore.path: /etc/elasticsearch/cacerts-java-extracted
-reindex.ssl.truststore.password: changeit
+# xpack.ssl.truststore.path: /etc/elasticsearch/cacerts-java-extracted
+# xpack.ssl.truststore.password: changeit
+# reindex.ssl.truststore.path: /etc/elasticsearch/cacerts-java-extracted
+# reindex.ssl.truststore.password: changeit
```

Comment:
@todo revisit this issue https://github.com/nasa/meza/commit/6d87712e95ba4b34aa5f013d96cc8904a196f9f2 because according to that, these lines are required - albeit suboptimal as a solution.